src/components/Channel/Channel.tsx

```diff
@@ -366,6 +367,16 @@ const ChannelInner = <
     skipMessageDataMemoization,
   } = props;
 
+  const channelQueryOptions: ChannelQueryOptions<StreamChatGenerics> & {
+    messages: { limit: number };
+  } = useMemo(
+    () =>
+      defaultsDeep(propChannelQueryOptions, {
+        messages: { limit: DEFAULT_INITIAL_CHANNEL_PAGE_SIZE },
+      }),
+    [propChannelQueryOptions],
+  );
```

Comment:
Use `DEFAULT_INITIAL_CHANNEL_PAGE_SIZE` when loading the first page of messages.